rds.tf

```diff
@@ -56,7 +57,7 @@ module "rds" {
   db_subnet_group_name = var.rds_db_subnet_group_name
   ca_cert_identifier   = var.rds_ca_cert_identifier
 
-  identifier = local.rds_identifier
+  identifier = lower(local.rds_identifier)
```

Comment:
but why lower?

Reply:
capslock problems? :D

Reply:
@ronny-panknin-flaconi because, the default DB that gets created on the instance is in uppercase. And [this](https://github.com/Flaconi/terraform-aws-microservice/pull/30/files#diff-28d019b21a3e2235030fc5e9d3e7163bL2-L3) is why this is needed.

Reply:
makes sense :)